### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by IhtishamMushtaq**
### Before hover
Screenshot from 2020-03-18 00-06-18

### After Hover
Screenshot from 2020-03-18 00-08-49

### Desktop and Tablet
Screenshot from 2020-03-18 00-11-44

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,35 +9,40 @@
...
+      <img class="logo" src="images/logo.png" alt="logo" />
+      <img
+        class="menu"
+        onfocus="countt() return;"
+        src="https://img.icons8.com/android/48/000000/menu.png"
+      />
+      <!-- <img
+        class="close"
+        src="https://img.icons8.com/carbon-copy/100/000000/close-window.png"
+      /> -->
```
Comment (Salman18) [inline index.html]: remove comments

File: index.html [hunk 2]
```diff
@@ -105,6 +116,6 @@ <h3>28-MAR-2020</h3>
         computer systems by looking for weaknesses and vulnerabilities in target
         systems, using the same knowledge and tools as a malicious hacker.
       </p>
-      <a href="">see more</a>
+      <a href="">see more</a> -->
```
Comment (Salman18) [inline index.html:116]: remove arrow -->

while resolving conflict be aware of these things

Comment (IhtishamMushtaq) [reply index.html:116]: Sir
i have done it deliberately
it is a comment

Comment (Salman18) [reply index.html:116]: `-->`

Comment (Salman18) [reply index.html:116]: resolve conflict

**Comment by IhtishamMushtaq** [comment]
resolved

**Review by Salman18 — APPROVED** [review]

**Pull request merged**